### ISSUE TITLE
"Open..."/"Import..." file choosers do not allow .pyui files

### ISSUE THREAD

**Issue by dgelessus** [omz/Pythonista-Issues#522]
Pythonista 3.2, tested on iOS 10 (by myself) and iOS 11 (by @metawops). When selecting "Open..." or "+" > "Import..." > "Files", the file chooser does not allow selecting .pyui files. However, the "Open In" menu for pyuis lists Pythonista as an option, and pyuis can be opened in-place when tapping them in the Files/iCloud Drive app.

**Comment by metawops** [comment]
However, when opening .pyui files in Pythonista by tapping on them in the iOS Files app they are not copied to Pythonista and thus not loadable/findable when you try to load them via ui.load_view.
A workaround for copying .pyui files into Pythonista is invoking the iOS Share Sheet on such a file (in the iOS Files app: long press, select Share from the context menu), select „Run Pythonista Script“, select the „Import File“ button in the top half of the Pythonista sheet.
Thanks to @dgelessus for pointing that out! 👍

**Comment by omz** [comment]
@metawops An alternative would be to use the “Import File” feature in the “new file” dialog.

**Comment by metawops** [comment]
Nope, unfortunately that doesn‘t work here because in the upcoming dialog .pyui files are greyed out, too. See screenshot.
![443b08ce-8fa7-422b-83d0-1c8a9033e9cd](https://user-images.githubusercontent.com/376090/35199071-ccc44354-fef8-11e7-90d6-dd7e17af41c4.png)

**Comment by metawops** [comment]
Additionally, not only are .pyui files not importable into Pythonista this way but also .md files from a github repository via Working Copy. See screenshot.
![ae1aa26b-c847-4d6b-b9c5-2a11ea6ea8ba](https://user-images.githubusercontent.com/376090/35295193-c32f2698-0078-11e8-8850-5bee9b0622d1.png)

**Comment by zrzka** [comment]
@metawops this is known issue, maybe some kind of limitation. As a workaround you can open whole folder / repo from WC.
